### ISSUE TITLE
Update focus indicator to have good contrast with background.

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#50]
The gold focus indicator does not meet sufficient contrast of 3:1 with the light gray background. 1.76:1 (#EAAA00 on #EEEEEE). Not a WCAG issue as it has a sufficient contrast witht he maroon linked text.
